src/main/java/seedu/modtrek/model/module/Code.java

```diff
@@ -9,7 +9,8 @@
 public class Code {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Module code is invalid. It should be alphanumeric.";
+            "Module code is invalid.\nIt should be in the form of <Module Prefix><4 number code><optional alphabet>.\n"
+                    + "Module Prefix should be 2 to 4 alphabets.\n" + "Examples: CS2109S, gess1025";
```

Comment:
Similar point as above.

Reply:
Noted with thanks. Added these changes to the new commit.